src/vma/util/utils.cpp

```diff
@@ -1035,6 +1035,37 @@ int validate_ipoib_prop(const char* ifname, unsigned int ifflags,
 	}
 }
 
+//NOTE RAW_QP_PRIVLIGES_PARAM_FILE does not exist on ofed upstream
```

Comment:
in comments, instead of '...ofed upstream' better to use '...upstream IB drivers'